packages/contracts/contracts/engine/V3/GenArt721CoreV3_Engine.sol

```diff
@@ -975,17 +1016,40 @@ contract GenArt721CoreV3_Engine is
             revert GenArt721Error(ErrorCodes.NewProjectsForbidden);
         }
         uint256 projectId = _nextProjectId;
-        projectIdToFinancials[projectId].artistAddress = _artistAddress;
+        ProjectFinance storage projectFinance = _projectIdToFinancials[
+            projectId
+        ];
+        projectFinance.artistAddress = _artistAddress;
         projects[projectId].name = _projectName;
         projects[projectId].paused = true;
         projects[projectId].maxInvocations = ONE_MILLION_UINT24;
         projects[projectId].projectBaseURI = defaultBaseURI;
+        // assign default artist royalty of 5% to artist
+        projectFinance
+            .secondaryMarketRoyaltyPercentage = _DEFAULT_ARTIST_SECONDARY_ROYALTY_PERCENTAGE;
```

Comment:
Should we accept `artistSecondaryRoyaltyPercentage` as a param?

Reply:
We could, but I wanted to minimize changes to creator dashboard. @Jbern16 - any thoughts on using default vs. adding param to the addProject function?

Reply:
We can add a param if you think it makes sense, changes would be minimal to support it. Could it fallback to default if param isn't provided? I also do wonder if changes like that belong in a major version update though, but I don't have a strong opinion either way.

Reply:
>Could it fallback to default if param isn't provided?
Only if we added a new external function so one with, one w/o the additional arg, and I don't think we have enough bytecode size margin to justify that

I think in the interest of keeping changes minimal (consider Engine partners with their own UIs as well as CD), I would rather keep this interface as unchanged. @lindsgil do feel more strongly? this could be something we configure in the initializer as well, so it could be defined per-contract.

Reply:
That sounds good to me! Good call out on configuring in the initializer 🙏

Reply:
cool, let's do that after PR #1514 to prevent the turbulence in current chain of PRs, and we can do some follow-on indexing work to index the default configured value 🙏